src/components/DisplayProductData.jsx

```diff
@@ -0,0 +1,43 @@
+import React, { Component } from "react";
+import { getData } from "../modules/productData";
+
+class DisplayProductData extends Component {
+  state = {
+    productData: [],
+  };
+
+  componentDidMount() {
+    this.getProductData();
+  }
+
+  async getProductData() {
+    let result = await getData();
+    this.setState({ productData: result.data.products });
```

Comment:
you can move the `data.products` to the module instead

Reply:
This should be fixed now!